web_widget_image_webcam/static/src/js/webcam_widget.js

```diff
@@ -0,0 +1,106 @@
+/*
+    Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+*/
+odoo.define('web_widget_image_webcam.webcam_widget', function(require) {
+    "use strict";
+
+    var core = require('web.core');
+    var Model = require('web.DataModel');
+
+    core.form_widget_registry.get("image").include({
+
+        render_value: function () {
+            this._super();
+
+            var self = this,
+                img_data;
+
+            // ::webcamjs:: < https://github.com/jhuckaby/webcamjs >
+            // Webcam: Set Custom Parameters
+            Webcam.set({
+                width: 320,
+                height: 240,
+                dest_width: 320,
+                dest_height: 240,
+                image_format: 'jpeg',
+                jpeg_quality: 90,
+                force_flash: false,
+                fps: 45,
+            });
+
+            self.$el.find('.oe_form_binary_file_clear').removeClass('col-md-offset-5');
+
+            new Model('ir.config_parameter').query(['key', 'value']).
+            filter([['key', '=', 'web_widget_image_webcam.flash_fallback_mode']]).
+            all().then(function(default_flash_fallback_mode) {
+                if (default_flash_fallback_mode.length && default_flash_fallback_mode[0]['value'] == 1) {
+                    Webcam.set({
+                        /*
+                            :: Important Note about Chrome 47+ :: < https://github.com/jhuckaby/webcamjs#important-note-for-chrome-47 >
+                            Setting "force_flash" to "true" will always run in Adobe Flash fallback mode on Chrome, but it is not desirable.
+                        */
+                        force_flash: true,
+                        /*
+                            If "force_flash" is true, then this extra parameter "swfURL" is necessary.
+                        */
+                        swfURL: '/web_widget_image_webcam/static/src/js/webcam.swf',
```

Comment:
Don't confuse enable_flash with force_flash.
https://github.com/jhuckaby/webcamjs#configuration
Flash is enabled by default, although just as a fallback. But in case the fallback gets used, then you need the `swfURL`.